tests/framework/cli/test_cli.py

```diff
@@ -517,22 +517,11 @@ def test_run_with_pipeline_filters(
         assert isinstance(runner, SequentialRunner)
         assert not runner._is_async
 
-    def test_with_sequential_runner_and_parallel_flag(
-        self, fake_project_cli, fake_session
-    ):
-        result = CliRunner().invoke(
-            fake_project_cli, ["run", "--parallel", "--runner=SequentialRunner"]
-        )
-        assert result.exit_code
-        assert "Please use either --parallel or --runner" in result.stdout
-
-        fake_session.return_value.run.assert_not_called()
-
     def test_run_successfully_parallel_via_flag(
```

Comment:
I think the name of this test can be updated too since it's not via flag anymore.

Reply:
Actually it turned out this test was now a duplicate of the one below it so I've removed it altogether.